web/src/app/core/resolvers/edit-project.resolver.ts

```diff
@@ -14,17 +14,17 @@ export class EditProjectResolver implements Resolve<boolean> {
         private authService: AuthenticationService,
         private projectService: ProjectService,
         private router: Router
-        ) {
+    ) {
     }
 
     resolve(route: ActivatedRouteSnapshot): any {
         this.projectService.findOneById(route.params.uid)
-        .subscribe((project: ProjectModel) => {
-            // Added check for not authenticated user and private project details as well for their ownner
-            if (project.type === 'private' && !this.authService.isAuthenticated ||
-               !project.access.admin.includes(this.authService.profile.uid)) {
-                this.router.navigate(['/']);
-            }
-        });
+            .subscribe((project: ProjectModel) => {
+                // Added check for not authenticated user and private project details as well for their ownner
+                if (project.type === 'private' && !this.authService.isAuthenticated ||
```

Comment:
This looks like 4 spaces?

Reply:
After configuration in settings of IDE to spaces 2 . Formatted all the code. 
Kindly review